advanced/Scripts/webpage.sh

```diff
@@ -393,13 +393,13 @@ ProcessDHCPSettings() {
         if [[ "${DHCP_LEASETIME}" == "0" ]]; then
             leasetime="infinite"
         elif [[ "${DHCP_LEASETIME}" == "" ]]; then
-            leasetime="24"
-            addOrEditKeyValPair "${setupVars}" "DHCP_LEASETIME" "${leasetime}"
+            leasetime="24h"
+            addOrEditKeyValPair "${setupVars}" "DHCP_LEASETIME" "24"
         elif [[ "${DHCP_LEASETIME}" == "24h" ]]; then
             #Installation is affected by known bug, introduced in a previous version.
             #This will automatically clean up setupVars.conf and remove the unnecessary "h"
-            leasetime="24"
-            addOrEditKeyValPair "${setupVars}" "DHCP_LEASETIME" "${leasetime}"
+            leasetime="24h"
+            addOrEditKeyValPair "${setupVars}" "DHCP_LEASETIME" "24"
```

Comment:
This entire section can be removed. There was one specific case in version 3 that had the possibility of `setupVars.conf` containing `DCHP_LEASETIME=24h` because we used to store that config with the time character. We originally stored it as seconds but people complained that math was hard and who knew what `86400` meant. 

We automatically added `h` to everything so in that specific case the `dnsmasq` configuration file ended up with `24hh` as the `leasetime`. I really don't think we need this shim anymore, no one should have `DCHP_LEASETIME=24h` in `setupVars.conf` now.

Keep it simple. If the var is `0` then it's infinite. If it's empty then `24h`, else it's the value of the var with `h` appended. You can not enter anything other than an integer via the web interface. 

The only other case I can see that should be accounted for is when the var contains anything other than an integer. In that case I think we use the default `24h`.

Reply:
Funilly enough, I was going to remove it - and then I backed out at the last minute. Should have gone with my gut and saved you the typing ;)